arangod/Cluster/Action.cpp

```diff
@@ -184,6 +184,17 @@ void Action::endStats() {
   _action->endStats();
 }
 
+void Action::failAll() {
```

Comment:
This seems redundant to block of code in MaintenanceWorker.cpp beginning at line 149.  Is this actually handling a known fail case that is not handled by MaintenanceWorker?